### ISSUE TITLE
Use .eslint rules to keep code better organized

### ISSUE THREAD

**Issue by bryon-czoch** [react-bootstrap-table/react-bootstrap-table2#222]
The code would be easier to read/contribute to if it was better organized.  

Here are some rules that would really help the readibilty:

   react/jsx-sort-props
   react/jsx-space-before-closing
   react/jsx-curly-spacing

**Comment by prajapati-parth** [comment]
Agreed with the 2nd and 3rd rule. But let's not consider `react/jsx-sort-props` for it's just an overhead when adding new props.

**Comment by AllenFang** [comment]
for the `jsx-space-before-closing` and `jsx-curly-spacing` just my favor styling lol

Anyway, I'm open for this, if it can help more people to join and contribute this project, let's fix it.

**Comment by AbdulKabia** [comment]
Hey, would it be okay if I were to work on this issue?
